app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -384,11 +384,13 @@ export const ContentForm: React.FC<IContentFormProps> = ({
                                 ).filter(
                                   (x) =>
                                     contentType !== ContentTypeName.Image ||
-                                    x.label.includes('(TC)') ||
-                                    x.label.includes('(PROVINCE)') ||
-                                    x.label.includes('(GLOBE)') ||
-                                    x.label.includes('(POST)') ||
-                                    x.label.includes('(SUN)'),
+                                    (typeof x.label === 'string'
```

Comment:
I'd love a cleaner way to do this, but for now it has to check if it's a component or string.